options/simple_options.cpp

```diff
@@ -141,7 +141,11 @@ simple_options::build_options()
                         "require signed key for new peers to join swarm")
                 (SIGNED_KEY.c_str(),
                         po::value<std::string>(),
-                        "signed key for node's uuid");
+                        "signed key for node's uuid")
+                (BLUZELLE_PUBLIC_KEY.c_str(),
```

Comment:
Weren't we going to NOT load from disc?

Reply:
We aren't, the public key is fead from the conf.